priv/repo/migrations/20190501221754_create_avatar_listings.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Ret.Repo.Migrations.CreateAvatarListings do
+  use Ecto.Migration
+
+  def change do
+    Ret.AvatarListing.State.create_type()
+
+    create table(:avatar_listings, prefix: "ret0", primary_key: false) do
+      add(:avatar_listing_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true)
+      add(:avatar_listing_sid, :string)
+      add(:slug, :string, null: false)
+      add(:order, :integer)
```

Comment:
i'd match it to scene listings, but without looking i'd have made it `default: 0, null: false`

Reply:
Yeah was thinking that, but scene_listings were declared like this, so I figured it would be cleaner to make them the same and change both later if we wanted to.